tests/unit/tab-bar.spec.js

```diff
@@ -0,0 +1,28 @@
+import { shallowMount, RouterLinkStub } from '@vue/test-utils';
+import Vue from 'vue';
+import TabBar from '../../src/popup/router/components/TabBar.vue';
+
+Object.assign(Vue.prototype, {
+  $t: () => 'locale-specific-text',
+});
+
+const routes = ['/account', '/balances', '/transfer', '/tips', { name: 'name-list' }];
+
+describe('TabBar', () => {
+  routes.forEach((route, index) => it('should have assigned route', () => {
+    const wrapper = shallowMount(TabBar, {
+      stubs: {
+        RouterLink: RouterLinkStub,
+      },
+    });
+    expect(wrapper.findAllComponents(RouterLinkStub).at(index).props().to).toStrictEqual(route);
+  }));
+  it('should have same amount of routes and RouterLinks', () => {
```

Comment:
In this case we would need to change this test everytime we would decide to change the routes in the component.